Services/source/housekeeping/housekeeping_to_beacon.c

```diff
@@ -0,0 +1,162 @@
+/*
+ * Copyright (C) 2015  University of Alberta
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU General Public License
+ * as published by the Free Software Foundation; either version 2
+ * of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ */
+/**
+ * @file housekeeping_to_beacon.c
+ * @author Grace Yi
+ * @date 2021-10-27
+ */
+#include "housekeeping_to_beacon.h"
+#include "housekeeping_service.h" //contains all housekeeping data
+#include "rtcmk.h" //to get time from RTC
+#include "services.h"
+#include "ex2_hal/ex2_adcs_software/equipment_handler/inc/adcs_handler.h"

```

Comment:
again, modify the project include path rather than use paths

Reply:
done

Reply:
Did you change this line?